### PR TITLE
Have the CI manage the docs

### PR DESCRIPTION
**Pull request by quba42**
`ci_update_docs: true` in `template_config.yml`

### CODE REVIEW

File: docs/changes.rst [hunk 1]
```diff
@@ -1,3 +1,5 @@
 .. _pulp_deb-changes:
 
 .. include:: ../CHANGES.rst
+
+.. include:: ../HISTORY.rst
```
Comment (quba42) [inline docs/changes.rst:5]: We don't have this file in the pulp_deb repo, but the plugin template wants to add it here. Will this cause problems?

Comment (mdellweg) [reply docs/changes.rst:5]: hmmm. Probably.

Comment (mdellweg) [reply docs/changes.rst:5]: An empty file seems to be fine though.

Comment (quba42) [reply docs/changes.rst:5]: Added an empty, appears to work.

**Review by hstct — APPROVED** [review]

**Pull request merged**